src/libraries/pkg/test/testPackages.proj

```diff
@@ -70,10 +72,12 @@
     <Copy SourceFiles="@(TestSupportFiles)" DestinationFolder="%(TestSupportFiles.DestinationFolder)" />
   </Target>
 
+  <UsingTask TaskName="GetCompatibilePackageTargetFrameworks" AssemblyFile="$(NuGetPackageRoot)\microsoft.dotnet.packagevalidation\$(MicrosoftDotNetPackageValidationVersion)\tools\net5.0\Microsoft.DotNet.PackageValidation.dll"/>
```

Comment:
Why not using the property that you exposed in the PackageValidation SDK's props file?

Reply:
Also is there a reason why this is hardcoded to net5.0 and there's no .NETFramework pendant?